ha/const.py

```diff
@@ -25,6 +25,9 @@
 SUPPORT_BUNDLE_LOGS=[RA_LOG_DIR, PCSD_LOG, PACEMAKER_LOG, COROSYNC_LOG]
 CORTX_SUPPORT_BUNDLE_LOGS=[RA_LOG_DIR]
 
+# Cluster Manager
+KVSTORE_VERSION="v1"
```

Comment:
Please name it as DATASTORE_VERSION. It will be applicable to all.

Reply:
Done